script/copy_deduplicate

```diff
@@ -135,6 +139,15 @@ parser.add_argument(
     default=0,
     help="Number of days preceding --date that should be used to filter out duplicates",
 )
+parser.add_argument(
+    "--billing_projects",
+    "--billing-projects",
+    "--billing_project",
+    "--billing-project",
+    nargs="+",
+    default=[None],
+    help="One or more billing projects over which bigquery jobs should be distributed",
```

Comment:
How does this interact with the project_id param? The docs on project_id (ID of the project in which to run query jobs) no longer apply.

Reply:
`--project_id` was never `(ID of the project in which to run query jobs)`, rather it was `ID of the project in which to find tables`. will update accordingly